araproc/analysis/spark_power_ratio.py

```diff
@@ -0,0 +1,102 @@
+# this looks for and identifies the spark events by calculating power ratio
+# and comparing with a set threshold 
+
+import numpy as np
+
+from araproc.framework import waveform_utilities as wfu
+
+
+
+
+
+def get_rms_power(waveform):
+  
+    """
+    Calculates RMS power of a waveform.
+
+    Parameters
+    ----------
+    waveform: TGraph
+        A TGraph of the waveform.
+
+    Returns
+    -------
+    power: float
+        RMS power of the waveform.
+    """
+
+    _, trace = wfu.tgraph_to_arrays(waveform)
+    power = np.nanmean(trace**2)
```

Comment:
we should drop the `np.nan*` functions. We want it to complain if a `nan` appears. So this can just be `np.mean`

Reply:
sure thing, I can fix that.